docs/manual.rst

```diff
@@ -2803,33 +2803,6 @@ used in many applications and you can expect that all operations are highly
 optimized. The creation of new geometries with many coordinates, however,
 involves some overhead that might slow down your code.
```

Comment:
This section above here will need to be rewritten a bit once the pygeos ufuncs are added to Shapely